db/models/Shirt.js

```diff
@@ -0,0 +1,70 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+
+const shirtSchema = new Schema({
+  searchID: { type: String, required: false },
+  pic1: {
```

Comment:
There feels like some repetition going on here. i.e. `pic1`, `pic2`, `pic3` and `pic4` all look very similar. 

We could move them to their own schema and then do something like this. 

``` js 
const picSchema = new Schema({
  picSRC: String,
  picSRCSlug: { type: String, required: false },
  picSRCSlugJPG: { type: String, required: false },
  variant1: { type: String, required: false },
  variant1slug: { type: String, required: false },
  variant2: { type: String, required: false },
  variant2slug: { type: String, required: false },
  variant3: { type: String, required: false },
  variant3slug: { type: String, required: false },
  variant4: { type: String, required: false },
  variant4slug: { type: String, required: false },
});

const shirtSchema = new Schema({
  searchID: { type: String, required: false },
  pic1: picSchema,
  pic2: picSchema,
  pic3: picSchema,
  pic4: picSchema,
  keywordOne: { type: String, required: true },
  keywordTwo: { type: String, required: true },
  keywordThree: { type: String, required: true },
});
```

Reply:
great idea! thank you!